components/brave_ads/browser/notification_helper_win.cc

```diff
@@ -69,34 +69,6 @@ bool NotificationHelperWin::ShouldShowNotifications() {
     LOG(WARNING) << "Notification not made: Full screen mode";
     return false;
   }
-
-  if (base::win::GetVersion() < base::win::Version::WIN10_RS4) {
-    // There was a Microsoft bug in Windows 10 prior to build 17134 (i.e.
-    // VERSION_WIN10_RS4) causing endless loops in displaying notifications. It
-    // significantly amplified the memory and CPU usage. Therefore, Windows 10
-    // native notifications in Chromium are only enabled for build 17134 and
-    // later
-    LOG(WARNING) << "Native notifications are not supported on Windows prior"
-        " to Windows 10 build 17134 so falling back to Message Center";
-    return true;
-  }
-
-  if (!base::FeatureList::IsEnabled(features::kNativeNotifications)) {
-    LOG(WARNING) << "Native notification feature is disabled so falling back to"
-        " Message Center";
-    return true;
-  }
-
-  if (IsFocusAssistEnabled()) {
-    LOG(INFO) << "Notification not made: Focus assist is enabled";
-    return false;
-  }
-
-  if (!IsNotificationsEnabled()) {
-    LOG(INFO) << "Notification not made: Notifications are disabled";
-    return false;
-  }
-
   return true;
 }
```

Comment:
`CanShowBackgroundNotifications` should be removed as custom notifications cannot be shown while the app is backgrounded

Reply:
done